jobsdb/jobsdb.go

```diff
@@ -84,6 +84,7 @@ type GetQueryParamsT struct {
 	StateFilters                  []string
 	JobCount                      int
 	EventCount                    int
+	PayloadSize                   *int64
```

Comment:
minor: `MaxPayloadSize` or `PayloadLimit` might be a better word choice than `PayloadSize`

Reply:
indeed
using `PayloadLimit`